src/content/docs/browser/browser-monitoring/page-load-timing-resources/cookie-collection-session-tracking.mdx

```diff
@@ -13,25 +13,25 @@ redirects:
   - /docs/browser/browser-monitoring/configuration/privacy-settings
 ---
 
-Browser monitoring's [page load timing](/docs/browser/new-relic-browser/page-load-timing-resources/page-load-timing-process) feature can track sessions by using [cookies](/docs/browser/new-relic-browser/page-load-timing-resources/new-relic-cookies-used-browser#jsessionid) that contain a simple session identifier.
+Browser monitoring's [page load timing](/docs/browser/new-relic-browser/page-load-timing-resources/page-load-timing-process) feature can track sessions by using the sessionStorage API that can store a simple session identifier.
 
 ## How it works [#process]
 
 Here is the basic process for session tracking:
 
-1. A session begins when a user visits a website where New Relic's browser monitoring is installed and cookie collection is enabled.
-2. After the user's browser is opened, our browser monitoring script sets a simple cookie identifying the session.
-3. The script then reads the cookie on subsequent page loads.
-4. The same session will continue until the user's browser is closed.
+1. A session begins when a user visits a website where New Relic's browser monitoring is installed and the privacy setting for tracking is enabled.
+2. As the user's page is loaded, our browser monitoring script sets a simple session identifier in the browser.
+3. The script then reads the ID on subsequent page loads, reloads and restores, within the same tab.
+4. The same session will continue until the user's tab/window is closed.
 
 Session tracking will not work properly in these situations:
 
-* If users have cookies disabled in their browser
-* If the browser is configured to **not** allow information associated with the New Relic cookie to be sent with the request from the monitored site
+* If users have DOM Storage disabled in their browser.
+* If the browser or page is configured to **not** allow loading and execution of third-party origin scripts, which naturally means the agent as a whole will not function, through security policies or other means.
```

Comment:
@cwli24 could we rearrange this sentence like this?

If the browser or page is configured to **not** allow loading and execution of third-party origin scripts through security policies or other means (which naturally means the agent as a whole will not function).

Reply:
Updated!

Reply:
Thanks!